lk2nd/device/dts/msm8909/msm8909-qrd-skub.dts

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8909 0>;
+	qcom,board-id = <QCOM_BOARD_ID_QRD 0x09>,
+			<QCOM_BOARD_ID(QRD, 1, 0) 0x09>;
+};
+
+&lk2nd {
+	lxf_p5001_b01 {
```

Comment:
I'd prefer to conform to linux code style for dts in which underscores are not used in names, you can drop the b01 suffix if you think that's more correct
```suggestion
	lxf-p5001-b01 {
```